docs/apis-tools/web-modeler-api/index.md

```diff
@@ -114,3 +129,21 @@ The API gives you access to the names, as well as the ids. For example, when req
 - **canonicalPath** contains the unique path. It is a list of **PathElementDto** objects which contain the id and the name of the element.
 
 Internally, the ids are what matters. You can rename files or move files between folders and projects and the id will stay the same.
+
+### How do I migrate from the `beta` API to the `v1` API? {#migrating-from-beta-to-v1}
+
+To migrate, change the base URL from `/api/beta` to `/api/v1`.
+
+**Breaking changes:**
```

Comment:
Do you feel this breaking change should be noted in any other Web Modeler documentation?

Regardless, I would probably note this as an admonition for attention.

Reply:
> Do you feel this breaking change should be noted in any other Web Modeler documentation?

I think it's fine to mention this here only. As It's cross-linked from the announcement page and 8.2 docs it is easily reachable by users.

> Regardless, I would probably note this as an admonition for attention.

Done, I wrapped the breaking changes into an admonition.